galore/__init__.py

```diff
@@ -113,20 +122,29 @@ def process_1d_data(input=['vasprun.xml'],
 
     broadened_data = data_1d.copy()
     if lorentzian:
-        broadened_data = galore.broaden(
-            broadened_data, d=d, dist='lorentzian', width=lorentzian)
+        broadened_data = galore.broaden(broadened_data,
+                                        d=d,
+                                        dist='lorentzian',
+                                        width=lorentzian)
 
     if gaussian:
-        broadened_data = galore.broaden(
-            broadened_data, d=d, dist='gaussian', width=gaussian)
+        broadened_data = galore.broaden(broadened_data,
+                                        d=d,
+                                        dist='gaussian',
+                                        width=gaussian)
 
     return (x_values, broadened_data)
 
 
 def process_pdos(input=['vasprun.xml'],
-                 gaussian=None, lorentzian=None,
```

Comment:
I found this easier to read before, I think autopep8 has got a bit carried away?